redwood/src/lib.rs

```diff
@@ -217,6 +217,39 @@ mod tests {
         assert_eq!(format!("{}", cert.fingerprint()), fingerprint);
     }
 
+    #[test]
+    fn test_source_key_creation_expiry() {
+        let (public_key, _secret_key, _fingerprint) =
+            generate_source_key_pair(PASSPHRASE, "foo@example.org").unwrap();
+        let cert = Cert::from_str(&public_key).unwrap();
+        let standard_creation_time = SystemTime::UNIX_EPOCH
+            // Equivalent to 2013-05-14
+            .checked_add(Duration::from_secs(1368507600))
```

Comment:
Since we're using the `1368507600` magic number twice now, let's refactor it into a constant?